plugins/zen-commerce/includes/class-rest-controller.php

```
@@ -0,0 +1,93 @@
+    }
+
+    public static function get_collections(WP_REST_Request $request): WP_REST_Response {
+        $lang  = $request->get_param('lang');
+        $limit = max(1, min(20, (int) $request->get_param('limit')));
+
+        $cache_key = 'zen_commerce_collections_v1_' . md5($lang . '|' . $limit);
+        $cached    = get_transient($cache_key);
+        if ($cached !== false) return rest_ensure_response($cached);
+
+        $response = [
+            'featured'     => Zen_Commerce_Product_Repository::query(['lang' => $lang, 'featured' => true, 'limit' => 1, 'orderby' => 'date', 'order' => 'DESC']),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Zen_Commerce_Product_Repository' in method 'get_collections'. (undefined)

(StaticAccess)